excel2sbol/tests/Test_converter_function_README.md

```diff
@@ -0,0 +1,32 @@
+These tests are to be updated as the code is updated.
+
+For file simple_library2.xlsx:
+
+Terms being tested:
+sbol_displayId
+sbol_roles
+sbh_designNotes
+sbh_alteredSequence
+sbol_description
```

Comment:
I really like this documentation. Very clear and helpful for later testing.

Reply:
you might just want to note as well that you are using ind and coldef sheets and that you have object types on the ind sheets.

Reply:
I'll put that in.